libs/usb/Types.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Types.h
+ * Types use with the USB subsystem.
+ * Copyright (C) 2015 Simon Newton
+ */
+
+#ifndef LIBS_USB_TYPES_H_
+#define LIBS_USB_TYPES_H_
+
+#include <utility>
+
+namespace ola {
+namespace usb {
+
+/**
+ * @brief Represents a device on the USB.
```

Comment:
This is another PAT Tester isn't it, but it still reads oddly. How about "Represents a USB device"?